logging/thirdparty/src/test/java/io/quarkus/ts/logging/jboss/SyslogIT.java

```diff
@@ -0,0 +1,62 @@
+package io.quarkus.ts.logging.jboss;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
```

Comment:
sorry @fedinskiy , I didn't mention before. could you add a tag with Quarkus ticket?

Reply:
Added to methods